### PR TITLE
MediaElement code style

### PR DESCRIPTION
**Pull request by pictos**
# For Reviewers

This PR is just to:
1. Adjust the code to match our code style;
2. Remove `sealed` from classes and add `protected` and `virtual` to some methods.

### Description of Change ###

This PR is to get MediaElement ready to be merged into the `main` branch.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

File: XamarinCommunityToolkit/Views/MediaElement/MediaElement.shared.cs [hunk 1]
```diff
@@ -205,27 +220,33 @@ void OnSourcePropertyChanged(MediaSource oldvalue, MediaSource newvalue)
...
+		{
+			var d = (double)newValue;
+
+			return d >= 0.0 && d <= 1.0;
+		}
 	}
 
-	[EditorBrowsable(EditorBrowsableState.Never)]
-	public class SeekRequested : EventArgs
+	class SeekRequested : EventArgs
```
Comment (pictos) [inline XamarinCommunityToolkit/Views/MediaElement/MediaElement.shared.cs:242]: The XF version that we target has this same Class, so for now I'll keep it here. When we upgrade the XF version for the project we can refactor this easily.

**Comment by pictos** [comment]
@ionixjunior can you review this PR? Let me what do you think

**Comment by ionixjunior** [comment]
@pictos of course! I'll check ASAP.

**Comment by ionixjunior** [comment]
@pictos I tested with my MediaElement sample project and everything worked out. In my sample project I have a problem in iOS renderer and now I could create a custom renderer of the MediaElement to solve it. Thank you!

**Comment by pictos** [comment]
tks @ionixjunior

**Comment by jfversluis** [comment]
> In my sample project I have a problem in iOS renderer and now I could create a custom renderer of the MediaElement to solve it. 

@ionixjunior is that something we need to fix in our code here? :)

**Comment by ionixjunior** [comment]
@jfversluis yes! I'll submit a PR for this.

**Pull request merged**